participant/app/root.tsx

```diff
@@ -84,6 +87,32 @@ export default function App() {
   // language, this locale will change and i18next will load the correct
   // translation files
   useChangeLanguage(locale);
+  const matomoUrl = MATOMO_URL_BASE || "http://localhost:8080/";
+
+  let isHydrated = useHydrated();
+  if (isHydrated) {
+    const tracker = new MatomoTracker({
+      urlBase: matomoUrl,
+      siteId: 1,
+      disabled: false, // optional, false by default. Makes all tracking calls no-ops if set to true.
+      heartBeat: {
+        // optional, enabled by default
+        active: true, // optional, default value: true
+        seconds: 10, // optional, default value: `15
+      },
+      linkTracking: false, // optional, default value: true
+      configurations: {
+        // optional, default value: {}
+        // any valid matomo configuration, all below are optional
+        disableCookies: true,
+        setSecureCookie: true,
```

Comment:
I think this might be good to set as an env var as well. Add `MATOMO_SECURE` to config.server.ts and add this:

```suggestion
        setSecureCookie: MATOMO_SECURE,
```